integration-tests/tests/workspace-dashboard.js

```diff
@@ -213,8 +213,11 @@ const testAzureWorkspace = withUserToken(async ({ page, token, testUrl }) => {
   // Verify tabs that currently depend on Google project ID are not present.
   await dashboard.assertTabs(['notebooks', 'workflows', 'job history'], false)
 
-  // Verify Analyses and Data tabs are present.
-  await dashboard.assertTabs(['analyses', 'data'], true)
+  // Verify Analyses tab is present.
+  // Data tab should only be visible in dev for Azure workspaces
+  const expectedTabs = environment === 'dev' ? ['data', 'analyses'] : ['analyses']
```

Comment:
Great! I thought alpha was also not "prod", but looking at the configuration files I see I am wrong.